src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
```

Comment:
Querying the DOM is an expensive operation. You are currently querying for the spider and wall on every click event. Consider querying for these elements outside of the event listener to improve performance.